### PR TITLE
Add toJUnitXML to test runner result classes

### PR DESCRIPTION
**Pull request by omusavi**
This change allows PrixFixe test runner changes to be consumed by
CI pipelines that use the JUnit XML format for recording test results

### CODE REVIEW

File: src/test_suite/test_suite.ts [hunk 1]
```diff
@@ -218,6 +254,26 @@ export class AggregatedResults {
         this.printLatencyStatistics();
     }
 
+    async toJUnitXml() {
```
Comment (drub0y) [inline src/test_suite/test_suite.ts]: I don't see any need for this to be `async` at the moment... are you future proofing the signature for async/io or something?

Comment (omusavi) [reply src/test_suite/test_suite.ts]: Good catch, development artifact...originally I would have it write to disk but then i changed my mind and had it return the XML instead

**Comment by MikeHopcroft** [comment]
Merged in aa99814e0f83c319ae9256fef231d902ab439e89

**Comment by MikeHopcroft** [comment]
Available in prix-fixe@v0.0.34

**Pull request closed without merging**